src/homework_8/myArrayList/MyArrayList.java

```diff
@@ -0,0 +1,138 @@
+package homework_8.myArrayList;
+
+import java.util.Arrays;
+
+public class MyArrayList<T> implements List<T> {
+    private static final Object[] EMPTY_ARRAY = {};
+    private static final Object[] FULL_ARRAY_CAPACITY = new Object[10];
+    private T[] array;
+
+    public MyArrayList() {
+        array = (T[]) FULL_ARRAY_CAPACITY;
```

Comment:
better write:
private static final int INITIAL_CAPACITY = 10; // in the class
array = (T[]) new Objects[INITIAL_CAPACITY]; // in constructor
and remove:
private static final Object[] FULL_ARRAY_CAPACITY = new Object[10];